src/Analyzer/Passes/FunctionToSubcolumnsPass.cpp

```diff
@@ -285,7 +285,7 @@ class FunctionToSubcolumnsVisitorFirstPass : public InDepthQueryTreeVisitorWithC
             return;
         }
 
-        if (const auto * join_node = node->as<JoinNode>())
+        if (const auto * /*join_node*/_ = node->as<JoinNode>())
```

Comment:
`clang19` change

Reply:
Would just `/*join_node*/` (no `_`) not be fine?

Reply:
![image](https://github.com/user-attachments/assets/94d0f7c4-967f-4a7e-b7b3-0aa6af4fa924)
Am I doing it wrong?